src/main/java/com/openlattice/chronicle/services/message/MessageManager.kt

```diff
@@ -0,0 +1,12 @@
+package com.openlattice.chronicle.services.message
+
+import java.util.*
+
+public interface MessageManager {
+    fun sendMessage(
+        organizationId: UUID,
+        studyId: UUID,
+        participantId: String,
+        messageDetails: Map<String, String>
```

Comment:
This should be `MessageDetails`, right?

```suggestion
        messageDetails: MessageDetails
```